src/measure_theory/measure/measure_space.lean

```diff
@@ -1943,6 +1948,36 @@ measure.ext (λ s hs, ext_on_measurable_space_of_generate_finite m0 C hμν le_r
 
 namespace measure
 
+section disjointed
+
+variable [measurable_space α]
```

Comment:
```suggestion
variable {m : measurable_space α}
```

Reply:
Is it a convention to always make `measurable_space` an implicit argument instead of an instance now?

Reply:
When the `measurable_space` argument can be deduced from a following `measure` argument, yes: this automatically gives versions that can be used even for other sigma-algebras than the main one, at virtually no cost.